src/55functions.js

```diff
@@ -157,7 +157,17 @@ stdlib.LOWER = stdlib.LCASE = function(s) {return und(s,'String(y).toLowerCase()
 //stdlib.LCASE = function(s) {return '('+s+').toLowerCase()';}
 
 
-// LTRIM
+// Returns a character expression after it removes leading blanks.
+// see https://docs.microsoft.com/en-us/sql/t-sql/functions/ltrim-transact-sql
+stdlib.LTRIM = function(s) {
+    return und(s, 'y.substr(y.indexOf(y.trim()), y.length)');
+}
+
+// Returns a character string after truncating all trailing spaces.
+// see https://docs.microsoft.com/en-us/sql/t-sql/functions/rtrim-transact-sql
+stdlib.RTRIM = function(s) {
+    return und(s, 'y.substr(0, y.indexOf(y.trim()) + y.trim().length)');

```

Comment:
I suggest you change to a solution based on regular expression. I believe something like the following should do the trick. 

``` 
y.replace(/\s+$/,"");
```